di_website/publications/models.py

```diff
@@ -471,6 +478,12 @@ class Meta:
 
     colour = models.CharField(max_length=256, choices=COLOUR_CHOICES, default=RED)
 
+    hide_date = models.BooleanField(
+        default=True,
+        verbose_name="Hide Related Links Date",
+        help_text="Should we hide the date on related links items ?",
+    )
```

Comment:
Dude? Why? ... I added my comment on a template file but u added a model field of it ... overkill ...

Reply:
I don't get it ... why would u implement a clearly terrible idea?? Since u misunderstood, you should have at least questioned my sanity cos this is a bad idea ... all I wanted was that template variable `content_type` to be renamed `hide_date` ... then I actually mentioned that "the only other change" would be in `related-content.html` ... I think you were the drunk one here.

Reply:
My bad, will get a second opinion next time. Thought you were refering to `configuration over code` so that users could change `hide_date` in the admin for each of the resources pages.